### PR TITLE
Multiple fixes for 2.18

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

[DYN-4930 - Update Group Defaults](https://jira.autodesk.com/browse/DYN-4930)

Screenshot 2023-04-07 at 2 35 40 PM

[DYN-5718 - Localize tooltip](https://jira.autodesk.com/browse/DYN-5718)

Screenshot 2023-04-07 at 2 38 36 PM

[DYN-5746 - Remove round edges](https://jira.autodesk.com/browse/DYN-5746)

Screenshot 2023-04-07 at 2 39 49 PM

[DYN-5751 - Format recommendation score](https://jira.autodesk.com/browse/DYN-5751)

Display `100%` properly, Min and Max width adjusted:
Screenshot 2023-04-07 at 4 10 35 PM

Display `<1%` for scores that are 0 or less than 1:
Screenshot 2023-04-07 at 4 14 32 PM

Unified score display in score tooltip, by using the same converter to format score in code-behind as well:
Screenshot 2023-04-07 at 4 38 05 PM

[DYN-5655 - Fix documentation link](https://jira.autodesk.com/browse/DYN-5655)

Removed an extra function with the same name and added label container:
Screenshot 2023-04-07 at 5 19 15 PM

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

- UI fixes

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

File: src/DynamoCoreWpf/Properties/Resources.resx [hunk 1]
```diff
@@ -1835,7 +1835,7 @@ Do you want to install the latest Dynamo update?</value>
     <value>Run started with new working range...</value>
   </data>
   <data name="GroupDefaultText" xml:space="preserve">
-    <value>&lt;Double click here to edit group description&gt;</value>
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Properties/Resources.resx:1838]: Are we using the same resource string in both Core and CoreWpf? Should we get rid of one of them?

File: src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml [hunk 2]
```diff
@@ -217,7 +217,7 @@
                                               Command="{Binding SignOutCommand}"
                                               Name="logoutOption">
                             <MenuItem.ToolTip>
-                                <ToolTip Content="Sign Out" Style="{StaticResource GenericToolTipLight}"/>
+                                <ToolTip Content="{x:Static p:Resources.LogoutMenuItemText}" Style="{StaticResource GenericToolTipLight}"/>
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml:220]: Thanks!

File: src/DynamoCoreWpf/Properties/Resources.resx (hunk 1 above)
Comment (zeusongit) [reply src/DynamoCoreWpf/Properties/Resources.resx:1838]: yes, we are using the same resource string in both, but the Wpf resources are unavailable in Core.

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs [hunk 3]
```diff
@@ -448,10 +448,10 @@ private void importTextBlock_MouseLeftButtonDown(object sender, MouseButtonEvent
...
         }
 
-        private void OnMoreInfoClicked(object sender, MouseButtonEventArgs e)
-        {
-            dynViewModel.OpenDocumentationLinkCommand.Execute(new OpenDocumentationLinkEventArgs(new Uri(Dynamo.Wpf.Properties.Resources.NodeAutocompleteDocumentationUriString, UriKind.Relative)));
-        }
+        //private void OnMoreInfoClicked(object sender, MouseButtonEventArgs e)
+        //{
+        //    dynViewModel.OpenDocumentationLinkCommand.Execute(new OpenDocumentationLinkEventArgs(new Uri(Dynamo.Wpf.Properties.Resources.NodeAutocompleteDocumentationUriString, UriKind.Relative)));
+        //}
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs:454]: @zeusongit Do we need to remove this commented code?

Comment (zeusongit) [reply src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs]: removed

**Pull request merged**